region/maridia/outer.json

```diff
@@ -6524,6 +6971,20 @@
                     "Only requires a runway of approximately 6 tiles in the adjacent room.",
                     "Perform an air ball while against the ceiling to make it up and through the morph tunnel."
                   ]
+                },
+                {
+                  "name": "G-Mode Morph IBJ",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2,3],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphIBJ",
```

Comment:
Should the non GMode strat require blind movement because Samus is covered for part of the IBJ?

Reply:
With GMode the camera is an issue for bringing morph into another room.  If you are ending GMode after reaching the top half of the room, you just need a way to know you are doine IBJing.

Reply:
I think the blind movement here is simple enough. I'll put a comment to help give a strategy for this.